packages/grant-explorer/src/features/round/__tests__/ViewRoundPage.test.tsx

```diff
@@ -1,58 +1,75 @@
-import { enableFetchMocks } from "jest-fetch-mock";
-
-enableFetchMocks();
-fetchMock.mockIf(/summary/, JSON.stringify({}));
-
 import ViewRound from "../ViewRoundPage";
 import { fireEvent, screen, waitFor } from "@testing-library/react";
 import {
   generateIpfsCid,
   makeApprovedProjectData,
   makeRoundData,
-  mockBalance,
-  mockNetwork,
-  mockSigner,
   renderWithContext,
 } from "../../../test-utils";
 import { faker } from "@faker-js/faker";
 import { Project, Round } from "../../api/types";
 import { payoutTokens } from "../../api/utils";
+import { vi } from "vitest";
+import { parseUnits, zeroAddress } from "viem";
 
-const chainId = 5;
-const roundId = faker.finance.ethereumAddress();
-const useParamsFn = () => ({ chainId: chainId, roundId: roundId });
-const userAddress = faker.finance.ethereumAddress();
-const mockAccount = {
-  address: userAddress,
-};
-const mockSwitchNetwork = {
-  chainId: chainId,
-};
-
-jest.mock("../../common/Navbar");
-jest.mock("../../common/Auth");
-jest.mock("@rainbow-me/rainbowkit", () => ({
-  ConnectButton: jest.fn(),
-}));
-
-jest.mock("wagmi", () => ({
-  useAccount: () => mockAccount,
-  useBalance: () => mockBalance,
-  useSigner: () => mockSigner,
-  useNetwork: () => mockNetwork,
-  useSwitchNetwork: () => mockSwitchNetwork,
-}));
-
-jest.mock("react-router-dom", () => ({
-  ...jest.requireActual("react-router-dom"),
-  useParams: useParamsFn,
-}));
+fetchMock.mockIf(/summary/, JSON.stringify({}));
```

Comment:
I noticed `msw` in `package.json`, are we using only one of `fetch-mock` and `msw` or both?

Reply:
i deprecated msw in favour of fetch-mock as it was only used in one page and was giving some trouble. I'd prefer we do everything thru msw, as it more closely reflects the real-world scenario. 
So here, it's jsut a leftover in package.json. Removed.